### ISSUE TITLE
warning gitbook-plugin-atoc@1.0.3: The engine "gitbook" appears to be invalid.

### ISSUE THREAD

**Issue by ad-m** [metabase/metabase#7558]
### Bugs
If you're filing an issue about a bug please include as much information
as you can including the following.

- Your browser and the version: not approciate
- Your operating system: Ubuntu 16.04.4 LTS
- Your databases: not approciate
- Metabase version: ```18bf85eb3f2642e7df9a198abd4d871af7176a05```
- Metabase hosting environment: Docker
- Metabase internal database: not approciate

- *Repeatable steps to reproduce the issue*

```bash
$ git clone https://github.com/metabase/metabase.git
...
$ cd metabase
$ docker build .
...
$ echo $npm_execpath | grep -q yarn || echo '\033[0;33mSorry, npm is not supported. Please use Yarn (https://yarnpkg.com/).\033[0m'
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
warning gitbook-plugin-atoc@1.0.3: The engine "gitbook" appears to be invalid.
info fsevents@1.1.3: The platform "linux" is incompatible with this module.
info "fsevents@1.1.3" is an optional dependency and failed compatibility check. Excluding it from installation.
[4/5] Linking dependencies...
warning " > postcss-cssnext@2.11.0" has unmet peer dependency "caniuse-db@^1.0.30000652".
[5/5] Building fresh packages...
Done in 105.58s.
Running 'webpack' with NODE_ENV=production assemble and minify frontend assets...
Node#before is deprecated. Use Node#raws.before
Hash: 64329f86ebc9f59a3a6f
Version: webpack 3.10.0
Time: 322690ms
...
```

**Comment by AlexMost** [comment]
Hey, that dependency was accidentally placed in  "dependencies" instead of "devDependencies" field in the package.json. Sorry for troubles, version `0.8.0` fixes that.

The main reason why we are moving to the new name is that [issue](https://github.com/ttag-org/ttag/issues/104). 

The migration process will require only renaming all occurrences of `c-3po` to `ttag`. We are also going to support the old `c-3po@0.8.x` version with bugfixes, so you can do that migration later.

Feel free to create issues with suggestions about how we can improve the translation workflow with `ttag`.

**Comment by ad-m** [comment]
I saw only a warning. I reported because the warning has its cause. Registering a warning as an issue is a step to eliminate the cause of the warning.

**Comment by flamber** [comment]
Fixed by #9733 since 0.32.9
